test/example-app/index.js

```diff
@@ -82,6 +82,12 @@ const legacyScriptingSource = `
         contacts[0].name = 'Patched by KEY_pre_poll & KEY_post_poll!';
         return contacts;
       },
+
+      contact_hook_scripting_catch_hook: function(bundle) {
```

Comment:
We should probably add another test here for returning an `array`.

Reply:
No problem! Added in eb0f462.